public/index.html

```diff
@@ -29,6 +29,7 @@
   <body>
     <noscript>You need to enable JavaScript to run this app.</noscript>
     <div id="root"></div>
+    <div id="modal-root"></div>
```

Comment:
why do we need this?

Reply:
As discussed, we are having a modal that is a fullscreen component overlaying on top of everything else. To do this, we need create an element at the top of the DOM so the component can be displayed on top. It's being used on TimerModal.jsx